docs/tls/README.md

```diff
@@ -13,7 +13,7 @@ Install MinIO Server using the instructions in the [MinIO Quickstart Guide](http
 
 ## <a name="use-an-existing-key-and-certificate-with-minio"></a>2. Use an Existing Key and Certificate with MinIO 
 
-This section describes how to use a private key and public certificate that have been obtained from a certificate authority (CA). If these files have not been obtained, skip to [3. Generate Self-signed Certificates](#generate-use-self-signed-keys-certificates) or generate them with [Let's Encrypt](https://letsencrypt.org) using these instructions: [https://docs.min.io/docs/generate-let-s-encypt-certificate-using-concert-for-minio](https://docs.min.io/docs/).
+This section describes how to use a private key and public certificate that have been obtained from a certificate authority (CA). If these files have not been obtained, skip to [3. Generate Self-signed Certificates](#generate-use-self-signed-keys-certificates) or generate them with [Let's Encrypt](https://letsencrypt.org) using these instructions: [Generate let's encypt certificate using concert for minio](https://docs.min.io/docs/generate-let-s-encypt-certificate-using-concert-for-minio.html).
```

Comment:
Is this supposed to be concert or CertBot?  The URL says concert but there is no reference to concert in the document

Reply:
There is also a typo in the url it says `encypt` instead of `encrypt` my PR is just fixing the link that sends you to the wrong place